src/Aspire.Cli/Program.cs

```diff
@@ -153,10 +155,15 @@ private static DirectoryInfo GetHivesDirectory()
 
     private static CliExecutionContext BuildCliExecutionContext(IServiceProvider serviceProvider)
     {
-        _ = serviceProvider;
+        var configuration = serviceProvider.GetRequiredService<IConfiguration>();
         var workingDirectory = new DirectoryInfo(Environment.CurrentDirectory);
         var hivesDirectory = GetHivesDirectory();
-        return new CliExecutionContext(workingDirectory, hivesDirectory);
+        
+        // Check if debug mode is enabled by looking at configuration or command line args
+        var debugMode = configuration.GetValue<bool>("debug") || 
```

Comment:
The change I am working on to expose the Command (and possibly the ParseResults) on the execution context means we won't need to do it this way.

Reply:
Do you want me to wait or merge ?